### PR TITLE
release 2.2.0

### PR DESCRIPTION
**Pull request by funky-eyes**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-seata-website/issues) filed for the change(usually before you start working on it).  Trivial changes such as typos do not necessitate a GITHUB issue. Your pull request should solely focus on addressing this specific issue without incorporating any other modifications. Each PR should resolve a single issue.
- [ ] Format the pull request title like `doc: add an operations guide document`. Each commit within the pull request should contain a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `npm run build`, and make sure it works as expected.
- [ ] Ensure that no files under the `build` folder are included in the commit, as well as the `package-lock.json` and `yarn.lock` files.

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs-unversioned/current/download/seata-server.md [hunk 1]
```diff
@@ -47,6 +49,7 @@ requirements, you may not be able to deploy Seata-Server on your device.
...
+| 2.2.0 | [apache-seata-2.2.0-incubating-src.zip](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-src.zip)<br/>[apache-seata-2.2.0-incubating-src.zip.asc](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-src.zip.asc)<br/>[apache-seata-2.2.0-incubating-src.zip.sha512](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-src.zip.sha512)<br/> | [apache-seata-2.2.0-incubating-bin.tar.gz](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-bin.tar.gz)<br/>   [apache-seata-2.2.0-incubating-bin.tar.gz.asc](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-bin.tar.gz.asc)<br/>   [apache-seata-2.2.0-incubating-bin.tar.gz.sha512](https://dist.apache.org/repos/dist/release/incubator/seata/2.2.0/apache-seata-2.2.0-incubating-bin.tar.gz.sha512)<br/> | [2.2.0 Release Notes](https://github.com/apache/incubator-seata/releases/tag/v2.2.0) | [2.2.0 Quick Start](/docs/user/quickstart/) |
```
Comment (slievrly) [inline i18n/en/docusaurus-plugin-content-docs-unversioned/current/download/seata-server.md:51]: usually put the newest one first.

File: i18n/en/docusaurus-plugin-content-docs/current/developers/ppmc-guide/release-guide_dev.md [hunk 2]
```diff
@@ -521,11 +527,78 @@ Checklist for reference:
...
+
+```
+Hi Incubator PMC,
+
+The vote to release Apache Seata(incubating) X.X.X-RCN has passed with
+3 +1 binding and 1 +1 non-binding votes, no +0 or -1 votes.
+
+Binding votes：
+
+- Huxing Zhang
```
Comment (slievrly) [inline i18n/en/docusaurus-plugin-content-docs/current/developers/ppmc-guide/release-guide_dev.md]: Here the name is best replaced by XXX.

Comment (funky-eyes) [reply i18n/en/docusaurus-plugin-content-docs/current/developers/ppmc-guide/release-guide_dev.md]: done

File: i18n/en/docusaurus-plugin-content-docs-unversioned/current/download/seata-server.md (hunk 1 above)
Comment (funky-eyes) [reply i18n/en/docusaurus-plugin-content-docs-unversioned/current/download/seata-server.md:51]: done

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**